src/streamlist/StreamList.js

```diff
@@ -34,6 +33,7 @@ export default class StreamList extends React.Component {
             description={showDescriptions && x.description}
             color={x.color}
             isSelected={x.name === selected}
+            isMuted={typeof x.in_home_view !== 'undefined' && !x.in_home_view}
```

Comment:
Ah ok. How about `x.in_home_view === false`?

Also, mind leaving a comment about how it can be undefined which means that we shouldn't show the mute icon?